makefile

```diff
@@ -96,8 +95,10 @@ ifneq ($(HOST_OS),Darwin)
 # Incompatible flags for clang on MacOS
 CFLAGS += \
 	-static-libgcc \
+	-static-libstdc++ \
 	-fstrict-volatile-bitfields \
-	-fno-tree-switch-conversion
+	-fno-tree-switch-conversion \
+	-fno-omit-frame-pointer
```

Comment:
Linux gets this twice? No harm, no foul.

Reply:
Ah, I took it out of the other spot anyway.